Tasks/DotNetCoreCLI/task.loc.json

```diff
@@ -14,11 +14,10 @@
     "Agent"
   ],
   "demands": [],
-  "preview": true,
   "version": {
     "Major": 2,
     "Minor": 1,
-    "Patch": 0
+    "Patch": 1
```

Comment:
Please bump up version for DotNetCoreInstaller task as well. I had removed preview flag from there earlier, but did not bump up patch version. Now since we are bumping up for all tasks, please do so for DotNetCoreInstaller as well.

Reply:
done